packages/webview_flutter/lib/src/webview_method_channel.dart

```diff
@@ -135,6 +135,7 @@ class MethodChannelWebViewPlatform implements WebViewPlatformController {
       'initialUrl': creationParams.initialUrl,
       'settings': _webSettingsToMap(creationParams.webSettings),
       'javascriptChannelNames': creationParams.javascriptChannelNames.toList(),
+      'autoMediaPlaybackPolicy': creationParams.autoMediaPlaybackPolicy.index,
```

Comment:
nit: Other plugins that deal with enums tend to use their stringified name instead of their ordinal. I think this is also OK, but it's a little less safe. There should at least be some kind of comment within `AutoMediaPlaybackPolicy` itself warning future devs not to re-arrange the order of enums, since it's the position and not the name that matters.

Reply:
Good point, I'd say it's worth choosing one way or another and add it to the style guide, though I guess it may not be needed if we adopt type safe method channels.

Note that tests will fail if a future dev reorders the fields so I'm not super worried about this.

Added a comment on the enum.